--- XaraLX-0.7r1766/xaralx.desktop.orig	2007-03-14 08:07:41.000000000 +0000
+++ XaraLX-0.7r1766/xaralx.desktop	2007-03-14 08:08:39.000000000 +0000
@@ -3,14 +3,15 @@
 Encoding=UTF-8
 Name=Xara Xtreme
 Comment=Versatile vector graphics program
-Comment[de_DE]=Leistungsstarkes Vektorgrafik-Programm
-Comment[fr_FR]=Logiciel d'illustration vectorielle polyvalent
-Comment[it_IT]=Versatile programma di grafica vettoriale
-Comment[pt_BR]=Versátil programa para gráficos vetoriais
-Icon=xaralx.png
+Comment[de]=Leistungsstarkes Vektorgrafik-Programm
+Comment[fr]=Logiciel d'illustration vectorielle polyvalent
+Comment[it]=Versatile programma di grafica vettoriale
+Comment[pl]=Wszechstronny program do grafiki wektorowej
+Comment[pt]=Versátil programa para gráficos vetoriais
+Icon=xaralx
 Terminal=false
 Type=Application
-Categories=Application;Graphics;VectorGraphics;
+Categories=GTK;Graphics;VectorGraphics;
 StartupNotify=true
 MimeType=application/vnd.xara;application/x-extension-xar;
 FilePattern=xaralx;
